### PR TITLE
Feat: Add job for transcoding and uploading imaging data

### LINKED ISSUE #76: Add jobs for transcoding + uploading imaging datasets
# User story
As a user, I want to be able to run a transcode job easily using a configuration file

# Acceptance criteria
- [ ] When a script is run, then the imaging acquisition folder will be copied to a destination location (s3, gcs, filesystem) with the same directory structure, except the raw data will be converted to OME-Zarr format.

# Sprint Ready Checklist 
1. - [ ] Acceptance criteria defined 
2. - [ ] Team understands acceptance criteria 
3. - [ ] Team has defined solution / steps to satisfy acceptance criteria 
4. - [ ] Acceptance criteria is verifiable / testable 
5. - [ ] External / 3rd Party dependencies identified
6. - [ ] Ticket is prioritized and sized

### PR DESCRIPTION
**Pull request by carshadi**
Closes #76 

Right now, the user has to be logged into `hpc-login` to run the job

### CODE REVIEW

File: src/aind_data_transfer/__init__.py [hunk 1]
```diff
@@ -8,4 +8,4 @@
 
 logging.basicConfig(format=LOG_FMT, datefmt=LOG_DATE_FMT)
 
-__version__ = "0.1.4"
+__version__ = "0.1.5"
```
Comment (jtyoung84) [inline src/aind_data_transfer/__init__.py:11]: I pushed a PR, so this needs to be 0.1.6 now.

File: scripts/gcs_upload.py [hunk 2]
```diff
@@ -129,7 +129,7 @@ def run_cluster_job(
 
 
 def run_python_local_job(
-    input_dir, bucket, gcs_path, n_workers=4, chunk_size=256 * 1024 * 1024
+    input_dir, bucket, gcs_path, n_workers=4, chunk_size=256 * 1024 * 1024, exclude_dirs=None
```
Comment (camilolaiton) [inline scripts/gcs_upload.py:132]: Documentation is outdated.

File: src/aind_data_transfer/transformations/compressors.py [hunk 3]
```diff
@@ -166,3 +166,31 @@ def scale_read_blocks(
...
+
+    class Compressors(Enum):
+        """Enum for compression algorithms a user can select"""
+
+        blosc = Blosc.codec_id
+
+    compressors = [member.value for member in Compressors]
+
+    @staticmethod
+    def get_compressor(compressor_name, kwargs):
```
Comment (camilolaiton) [inline src/aind_data_transfer/transformations/compressors.py:181]: In this context, should the parameter kwargs have double pointers to include n parameters that will be passed into Blosc?

**Review by camilolaiton — APPROVED** [review]

File: src/aind_data_transfer/transformations/compressors.py (hunk 3 above)
Comment (carshadi) [reply src/aind_data_transfer/transformations/compressors.py:181]: good catch, will fix

**Pull request merged**